src/discretization/generate_bc_eqs.jl

```diff
@@ -105,6 +105,55 @@ function generate_bc_eqs(s::DiscreteSpace{N,M,G}, boundaryvalfuncs, boundary::Ab
     end)
 end
 
+"""
+`generate_extrap_eqs`
+
+Pads the boundaries with extrapolation equations, extrapolated with 6th order lagrangian polynomials.
```

Comment:
Why 6th?

Reply:
To maintain 5th order accuracy for weno, it is used in the spec.